.github/workflows/on-push.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           echo "NODE_ENV=test" >> .env
           echo "APP_NAME=jira" >> .env
           echo "APP_ID=${{ secrets.E2E_GITHUB_APP_ID }}" >> .env
-          echo "WEBHOOK_SECRET=${{ secrets.E2E_GITHUB_WEBHOOK_SECRET }}" >> .env
+          echo "WEBHOOK_SECRETS=${{ secrets.E2E_GITHUB_WEBHOOK_SECRET }}" >> .env
```

Comment:
I would probably just add a new secret here called webhook_secrets.  Did you actually change the value of of the secret?  Because if you did, you just broke all e2e tests for every other runs that isn't on this branch.

Reply:
Yep, will create a new one and update this. 
However, this one is a bit of oddball, I haven't changed anything here. Yet it passes the test cases.